src/utils.js

```diff
@@ -1,4 +1,10 @@
+let log = console.log;
```

Comment:
As discussed, in this case log function will be global, and creating two services with custom log functions will cause an error.

Me and @shmuga are now discussing another possible ways to restructure services, thus avoiding issue listed above